client/utils.go

```diff
@@ -152,17 +153,24 @@ func authortext(AuthorText sql.NullString, authors []database.Author) string {
 	var sb strings.Builder
 	sb.WriteString("Skriven av ")
 
-	sb.WriteString(authorsName(authors[0]))
+	sb.WriteString(string(authorURL(authorsName(authors[0]), authors[0].KthID)))
 	if len(authors) == 1 {
-		return sb.String()
+		return template.HTML(sb.String())
```

Comment:
You're only supposed to cast a string to a `template.HTML` if that is supposed to be interpreted as html, right? Are the author's names supposed to be interpreted as such?